consultation_analyser/consultations/urls.py

```diff
@@ -17,10 +25,37 @@
         answers.index,
         name="question_responses",
     ),
+    # New modular endpoints
+    # API endpoints
     path(
-        "consultations/<str:consultation_slug>/responses/<str:question_slug>/json/",
-        answers.question_responses_json,
-        name="question_responses_json",
+        "api/consultations/<str:consultation_slug>/questions/<str:question_slug>/demographic-options/",
+        DemographicOptionsAPIView.as_view(),
+        name="api_demographic_options",
+    ),
+    path(
+        "api/consultations/<str:consultation_slug>/questions/<str:question_slug>/demographic-aggregations/",
+        DemographicAggregationsAPIView.as_view(),
+        name="api_demographic_aggregations",
+    ),
+    path(
+        "api/consultations/<str:consultation_slug>/questions/<str:question_slug>/theme-information/",
+        ThemeInformationAPIView.as_view(),
+        name="api_theme_information",
+    ),
+    path(
+        "api/consultations/<str:consultation_slug>/questions/<str:question_slug>/theme-aggregations/",
+        ThemeAggregationsAPIView.as_view(),
+        name="api_theme_aggregations",
+    ),
+    path(
+        "api/consultations/<str:consultation_slug>/questions/<str:question_slug>/filtered-responses/",
```

Comment:
```suggestion
        "api/consultations/<str:consultation_slug>/questions/<str:question_slug>/responses/",
```

let the "/?field=value" signify that this is filtered